elytron-oidc-client/src/main/resources/schema/wildfly-elytron-oidc-client_2_0.xsd

```diff
@@ -799,21 +799,15 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="realm" type="xs:string" minOccurs="0" maxOccurs="1">
+            <xs:element ref="realm-or-provider" minOccurs="0" maxOccurs="1">
```

Comment:
This would be better modeled as an optional xs:choice containing the realm and provider elements.